optaplanner-core/src/main/java/org/optaplanner/core/impl/localsearch/decider/acceptor/greatdeluge/GreatDelugeAcceptor.java

```diff
@@ -0,0 +1,93 @@
+package org.optaplanner.core.impl.localsearch.decider.acceptor.greatdeluge;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.impl.localsearch.decider.acceptor.AbstractAcceptor;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchMoveScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchPhaseScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchStepScope;
+import org.optaplanner.core.impl.score.ScoreUtils;
+
+public class GreatDelugeAcceptor extends AbstractAcceptor {
+
+    private Score initialWaterLevel = null;
+    private Score currentWaterLevel = null;
+
+    private Score waterLevelIncrementScore = null;
+    private Double waterLevelIncrementRatio = DEFAULT_WATER_LEVEL_INCREMENT_RATIO;
+
+    // Good value to come out from. Source: https://github.com/UniTime/cpsolver from Tomas Muller
+    private static final double DEFAULT_WATER_LEVEL_INCREMENT_RATIO = 0.00_000_005;
+
+
+    public Score getWaterLevelIncrementScore() {
+        return this.waterLevelIncrementScore;
+    }
+
+    public void setWaterLevelIncrementScore(Score waterLevelIncrementScore) {
+        this.waterLevelIncrementScore = waterLevelIncrementScore;
+        this.waterLevelIncrementRatio = null;
```

Comment:
Use a simple setter (notice that the setWaterLevelIncrementRatio doesn't have this side effect) and have either the AcceptorConfig set the default value, similar to how it sets 400 for Late Acceptance.